### PR TITLE
Update ILB e2e test resource names

### PR DESCRIPTION
**Pull request by spencerhance**
Change the names of the resources in the ilb e2e tests to make debugging easier

### CODE REVIEW

**Comment by spencerhance** [comment]
@MrHohn fixed the test failure

File: cmd/e2e-test/ilb_test.go [hunk 1]
```diff
@@ -39,6 +39,11 @@ var (
 func TestILB(t *testing.T) {
 	t.Parallel()
 
+	// These names are useful when reading the debug logs
+	testName := "test-ilb-basic"
```
Comment (bowei) [inline cmd/e2e-test/ilb_test.go]: probably want to match testName with "TestILB"

Comment (spencerhance) [reply cmd/e2e-test/ilb_test.go]: Only reason I didn't do that here was that every other testName starts with "test-ilb," so this test wouldn't be easily searchable.

**Pull request merged**